src/main/java/com/revature/services/impl/DistanceServiceImpl.java

```diff
@@ -1,145 +1,86 @@
 package com.revature.services.impl;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
-
 import com.google.maps.DistanceMatrixApi;
 import com.google.maps.DistanceMatrixApiRequest;
 import com.google.maps.GeoApiContext;
 import com.google.maps.errors.ApiException;
 import com.google.maps.model.DistanceMatrix;
 import com.google.maps.model.TravelMode;
 import com.google.maps.model.Unit;
+import com.revature.models.Address;
 import com.revature.models.User;
 import com.revature.services.DistanceService;
 import com.revature.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.*;
 
 @Service
 public class DistanceServiceImpl implements DistanceService {
-	
+
 	@Autowired
 	private UserService us;
 
+	//TODO: REFACTOR THIS
```

Comment:
If that has already been refactored, please remove this.

Reply:
That hasn't been refactored yet.